src/coord/src/coord.rs

```diff
@@ -2594,6 +2594,7 @@ where
             session
                 .vars()
                 .iter()
+                .filter(|v| v.name() != "qgm_optimizations")
```

Comment:
We should not hardcode these. Instead, add a new `hidden() -> bool` method to the `Var` trait and filter on that, probably implemented as a bool on `ServerVar`

Reply:
I added `experimental` variable instead and changed the check to `!v.experimental()`.